lib/package.gi

```diff
@@ -225,36 +225,30 @@ BindGlobal( "AddPackageInfos", function( files, pkgdir, ignore )
         NormalizeWhitespace( pkgname );
         version:= record.Version;
 
-        # If we have this version already then leave it out.
-        if ForAll( GAPInfo.PackagesInfo,
-                    r ->    r.PackageName <> record.PackageName
-                        or r.Version <> version ) then
```

Comment:
Personally, I prefer code which is not deeply nested, makes it easier for me to keep the state in mind. Here, that could be achieved by negating the condition and using `continue;` in the body of the `if`, no? Then the diff would also be minimized.

But that's in the end a matter of taste.

Reply:
I do not understand this comment.
The point is that this `if` disappears completely.

Reply:
For some reason the comment ended up in the wrong line. I'll move it.

Reply:
Ah no, in fact, it was indeed just nonsense, I had turned on "ignore whitespace" and then apparantly develop a temporary case of blindness / stupidity :-). Sorry for the confusion